hack/test-integration.sh

```diff
@@ -33,7 +33,15 @@ if [ "$COVER" -eq 1 ]; then
 fi
 
 # Run the tests and collate the results.
-bats -t $ROOT/test/*.bats
+tests=""
+if [[ -z "${TESTS}" ]]; then
+  tests=${ROOT}/test/*.bats
+else
+  for f in ${TESTS:-*}; do
```

Comment:
The `:-*` is unnecessary here. Also, if you could use an array here rather than a string that'd be great. Something like:

```
tests=()
if [ -z "$TESTS" ]; then
  tests=( $ROOT/test/*.bats )
else
  for f in $TESTS; do
    tests+=( "$ROOT/test/$f.bats")
  done
fi
```

Also, tabs > spaces. :wink: